src/wp-includes/post.php

```diff
@@ -558,6 +558,36 @@ function create_initial_post_types() {
 		)
 	);
 
+	register_post_type(
+		'wp_font_family',
+		array(
+			'label'        => _x( 'Font Family', 'post type general name' ),
+			'description'  => __( 'Font Family definition for installed fonts.' ),
+			'public'       => false,
+			'_builtin'     => true, /* internal use only. don't use this when registering your own post type. */
+			'_edit_link'   => '/site-editor.php?canvas=edit', /* internal use only. don't use this when registering your own post type. */
+			'show_ui'      => false,
+			'show_in_rest' => true,
```

Comment:
This post type shows up in the rest api? What is the endpoint used for?

Reply:
To get the font families on the client side using `useEntityRecords()` .

Reply:
@swissspidy I followed the example of global styles, which posts have `wp_global_styles` as post type.

Reply:
@matiasbenedetto For the post type, the name is okay, but I think what @swissspidy is referring to is the slug _specifically_ used for the REST API.

Reply:
These was changes in the latests commits, we are now using these endpoints:

#### Font families:
POST /font-families/
DELETE /fonts-families/<slug>
GET /fonts-families
GET /fonts-families/<slug>

#### Font collections:
GET /fonts-collections
GET /fonts-collections/<id>